src/patternfly/components/Select/select-toggle-typeahead.hbs

```diff
@@ -5,9 +5,17 @@
     form-control--attribute=(concat 'type="text" id="' id '-typeahead" aria-label="Type to filter" value="' select-typeahead--Placeholder '"')}}
   {{/form-control}}
 {{else}}
-  {{#> form-control controlType="input"
-    input="true"
-    form-control--modifier="pf-c-select__toggle-typeahead"
-    form-control--attribute=(concat 'type="text" id="' id '-typeahead" aria-label="Type to filter" placeholder="' select-typeahead--Placeholder '"')}}
-  {{/form-control}}
+  {{#if select--IsDisabled}}
+    {{#> form-control controlType="input"
+      input="true"
+      form-control--modifier="pf-c-select__toggle-typeahead pf-m-disabled"
```

Comment:
Same here - since this renders as an `<input>` element, we want to use the `disabled` attribute instead of the `pf-m-disabled class`.

Reply:
I had used both the `pf-m-disabled` class and the `disabled` attribute for both of these, and it rendered correctly.  I got right of the `disabled` attributes on both the `<input>` and `<button>` because this seemed to be causing the accessibility test failures.

Reply:
Ah gotcha. Looking at [this commit](https://github.com/patternfly/patternfly/pull/2829/commits/32a62317ccb6d2a6fba32c199e976a19d96d41f2) it looks like you replaced all of `form-control--attribute` with `disabled` via `form-control--attribute="disabled"`. Instead what you should do is take the existing value of `form-control--attribute` and add ` disabled ` to it somewhere between other attributes. For example, between `type="text" id="` as `type="text" disabled id="`